lib/acts_as_span.rb

```diff
@@ -34,6 +36,14 @@ def acts_as_span(*args)
 
       options = OpenStruct.new(args.last.is_a?(Hash) ? ActsAsSpan.options.merge(args.pop) : ActsAsSpan.options)
 
+      unsupported_options =
+        options.to_h.keys.reject { |opt| OPTIONS.include? opt }
+      unless unsupported_options.empty?
+        raise ArgumentError,
+          'Unsupported option(s): ' \
+          "#{unsupported_options.map { |o| "'" << o.to_s << "'" }.join(', ')}"
```

Comment:
Why not use interpolation? 
```suggestion
          "#{unsupported_options.map { |o| "'#{o}'" }.join(', ')}"
```

Reply:
Because it felt weird to have an interpolation inside an interpolation, but the parser seems to have no trouble understanding it, so I'm happy to roll with it.